docs/api/security/trusted_storage.md

```diff
@@ -0,0 +1,27 @@
+<h2 id="blockdevice-port">Trusted Storage Reference API</h2>
```

Comment:
this line probably copied from elsewhere and is not rellevant

Reply:
It is the page headline. It says this is the Reference API page.
The section explains about the ITS and has its own headline.
I will make a small change here